### PR TITLE
Add Vietnamese translation

### PR DESCRIPTION
**Pull request by CactiStaccingCrane**
Add even more Vietnamese strings

### CODE REVIEW

File: website/public/locales/vi/common.json [hunk 1]
```diff
@@ -2,15 +2,25 @@
   "about": "Thông tin",
   "account_settings": "Tài khoản",
   "admin_dashboard": "Trang cho admin",
+  "back_to_dashboard": "Quay về trang chính",
+  "bug_description": "Nếu bạn tự dưng vào trang này thì nhớ báo lỗi:",
+  "bug_button": "Báo lỗi",
+  "cancel": "Bỏ",
```
Comment (notmd) [inline website/public/locales/vi/common.json:8]:
```suggestion
  "cancel": "Hủy",
```

File: website/public/locales/vi/common.json [hunk 2]
```diff
@@ -2,15 +2,25 @@
...
+  "create_chat": "Tạo cuộc trò chuyện mới",
   "dark_mode": "Giao diện tối",
-  "dashboard_home": "Trang chính",
   "dashboard": "Trang chính",
+  "dashboard_home": "Trang chính",
   "delete": "Xoá",
   "discord": "Discord",
-  "docs": "Hướng dẫn",
+  "docs": "Tài liệu",
+  "edit": "Sửa",
```
Comment (notmd) [inline website/public/locales/vi/common.json:21]:
```suggestion
  "edit": "Chỉnh sửa",
```

File: website/public/locales/vi/labelling.json [hunk 3]
```diff
@@ -2,7 +2,7 @@
   "accurate": "Chính xác",
   "apolitical": "Không có yếu tố chính trị",
   "appropriate": "Thích hợp",
-  "boring": "Nhạt nhẽo",
+  "boring": "Nhạt",
```
Comment (notmd) [inline website/public/locales/vi/labelling.json:5]: I'm not sure "Nhạt" is good here. Maybe something like "Nhàm chán"?

File: website/public/locales/vi/labelling.json [hunk 4]
```diff
@@ -38,7 +38,7 @@
   "label_message_yes_no_instruction": "Trả lời các câu hỏi sau:",
   "lang_mismatch": "Sai ngôn ngữ",
   "lang_mismatch.explanation": "Tin nhắn không được viết trong ngôn ngữ được chọn.",
-  "low_quality": "Chất lượng thấp",
+  "low_quality": "Không ổn",
```
Comment (notmd) [inline website/public/locales/vi/labelling.json:41]: I think the old translation is better.

File: website/public/locales/vi/message.json [hunk 5]
```diff
@@ -1,16 +1,20 @@
 {
+  "confirm_open_link_body": "Bạn có muốn mở tin nhắn này không?",
```
Comment (notmd) [inline website/public/locales/vi/message.json:2]:
```suggestion
  "confirm_open_link_body": "Bạn có muốn mở đường dẫn này không?",
```

File: website/public/locales/vi/message.json [hunk 6]
```diff
@@ -1,16 +1,20 @@
...
+  "confirm_open_link_header": "Xác nhận mở tin nhắn",
```
Comment (notmd) [inline website/public/locales/vi/message.json:3]:
```suggestion
  "confirm_open_link_header": "Xác nhận mở đường dẫn",
```

File: website/public/locales/vi/tasks.json [hunk 7]
```diff
@@ -58,28 +52,39 @@
...
+    "label": "Phân loại các tin nhắn của Open Assistant",
+    "desc": "Tạo nhãn dữ liệu để đánh giá tin nhắn.",
+    "overview": "Từ cuộc trò truyện ở dưới, trả lời các câu hỏi về câu trả lời cuối trong cuộc trò truyện."
+  },
+  "submitted_as": "Câu trả lời sẽ được gửi đi là ngôn ngữ {{submit_lang}}",
+  "tab_write": "Viết",
+  "tab_preview": "Xem trước",
   "writing_wrong_langauge_a_b": "Tin nhắn có vẻ được viết trong ngôn ngữ {{detected_lang}} nhưng sẽ được gửi đi là ngôn ngữ {{submit_lang}}.",
-  "submitted_as": "Câu trả lời sẽ được gửi đi là ngôn ngữ {{submit_lang}}"
+  "not_rankable": "Tất cả các câu trả lời đều sai và không thể phân loại được"
```
Comment (notmd) [inline website/public/locales/vi/tasks.json:89]:
```suggestion
  "not_rankable": "Tất cả các câu trả lời đều sai và không thể xếp hạng được"
```

File: website/public/locales/en/tasks.json [hunk 8]
```diff
@@ -20,7 +20,7 @@
   },
   "reply_as_user": {
     "label": "Reply as User",
-    "desc": "Chat with Open Assistant and help improve its responses as you interact with it.",
+    "desc": "Chat with Open Assistant and help improve it's responses as you interact with it.",
```
Comment (notmd) [inline website/public/locales/en/tasks.json:23]: https://github.com/LAION-AI/Open-Assistant/pull/2037 made this change. As I'm not a native speaker, I'm not sure which is correct

File: website/public/locales/vi/message.json [hunk 9]
```diff
@@ -1,16 +1,20 @@
...
   "message": "Tin nhắn",
   "message_author": "Viết bởi bạn",
   "message_author_explain": "Bạn là tác giả của tin nhắn này",
   "message_deleted": "Tin nhắn đã xoá",
+  "no_messages": "Không có tin nhắn",
   "open_new_tab_action": "Mở ở trang mới",
   "parent": "Tin nhắn gốc",
-  "reactions": "Bình luận",
-  "recent_messages": "Tin nhắn gần đây",
+  "reactions": "Biểu tượng",
```
Comment (notmd) [inline website/public/locales/vi/message.json:16]:
```suggestion
  "reactions": "Biểu tượng cảm xúc",
```

File: website/public/locales/en/tasks.json (hunk 8 above)
Comment (CactiStaccingCrane) [reply website/public/locales/en/tasks.json:23]: it's a typo, sorry.

**Review by AbdBarho — APPROVED** [review]
Thanks!

**Pull request merged**